src/main/modules/properties-volume/index.ts

```diff
@@ -8,7 +8,7 @@ export class PropertiesVolume {
     if (server.locals.ENV !== 'development') {
       propertiesVolume.addTo(config);
 
-      this.setSecret('secrets.rpe.AppInsightsInstrumentationKey', 'appInsights.instrumentationKey');
+      this.setSecret('secrets.bulk-scan-helper-frontend.AppInsightsInstrumentationKey', 'appInsights.instrumentationKey');
```

Comment:
We don't need this actually, as we are not plugging into app insights

Reply:
removed